index.md

```diff
@@ -46,9 +46,21 @@ Taxonomic classifiers perform best when they are trained based on your specific
 ```
 
 ```{card}
-:header: **human stool weighted Silva 138 99% OTUs full-length sequences**
+:header: **diverse weighted Silva 138 99% OTUs full-length sequences**
 
-**Download**: [human stool weighted Silva 138 99% OTUs full-length sequences](https://data.qiime2.org/classifiers/sklearn-1.4.2/silva/silva-138-99-nb-human-stool-weighted-classifier.qza)\
+**Download**: [diverse weighted Silva 138 99% OTUs full-length sequences](https://data.qiime2.org/classifiers/sklearn-1.4.2/silva/silva-138-99-nb-diverse-weighted-classifier.qza)\
+**UUID**: eff4efb9-d90d-43ce-acb3-53e04583323a\
+**SHA256**: decfae408061fab8ff2fec7dac1fe2a2e0041581589715062cc789bd4f9933db\
+**Sklearn Version**: 1.4.2\
+**Date Trained**: 2024-07-04\
+**Notes**: [Silva species taxonomy may be unreliable](#Silva)\
```

Comment:
```suggestion
**Notes**: [Silva species taxonomy may be unreliable. These classifiers were created with 14 diverse environments: "Sediment (non-saline)", "Plant corpus", "Animal secretion", "Sediment (saline)", "Animal surface", "Surface (saline)",  "Plant rhizosphere",  "Soil (non-saline)", "Animal distal gut", "Water (saline)",  "Animal proximal gut", "Water (non-saline)",  "Animal corpus", "Plant surface". More information can be found at: https://www.nature.com/articles/s41467-019-12669-6#Sec7 ](#Silva)\
```

Reply:
We also need to add the clawback citation: https://www.nature.com/articles/s41467-019-12669-6